meson.build

```diff
@@ -0,0 +1,40 @@
+project('spp', 'c')
+
+platform_inc = ['.']
+c_args = []
+
+spp_files = [
+  'spp.c'
+]
+
+if get_option('use_r_util') and get_option('r_util_includes') != ''
```

Comment:
im not sure about the point of this option, r_api.c is required to build for spp.c and it's definively confusing to have a subproject to have an option for a superproject.

Reply:
r_api is not required. Unfortunately SPP is a bit problematic. If you compile it standalone, r_api.c is used to define r_strbuf_*, r_sys_* and other used functions.

But when used inside radare2, r_strbuf_* and r_sys* functions are provided by r_util.